.github/workflows/publish.yml

```diff
@@ -46,3 +46,9 @@ jobs:
           dotnet nuget push ./artifacts/*.nupkg \
             --api-key $NUGET_API_KEY \
             --source https://api.nuget.org/v3/index.json
+
+      - name: Push Symbol package to NuGet
```

Comment:
I see this isn't needed

![image](https://github.com/user-attachments/assets/74b96f10-c37a-4b66-88cc-5ea8d6d0529a)
https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg

will revert that

Reply:
fixed